cli/foundation-models/system/finetune/stable_diffusion/deploy-online.yaml

```diff
@@ -0,0 +1,12 @@
+$schema: https://azuremlschemas.azureedge.net/latest/managedOnlineDeployment.schema.json
```

Comment:
Instead of cli/foundation-models/system/finetune/stable_diffusion; let's have cli/foundation-models/system/finetune/text-to-image